qcut/docs/issues/virtual-folder-implementation-plan.md

```
@@ -0,0 +1,1174 @@
+    <ExternalLink className="h-4 w-4 mr-2" />
+    Open in Explorer
+  </ContextMenuItem>
+)}
+```
+
+---
+
+## File Path Summary
+
+### New Files
+| File | Purpose |
```

markdownlint-cli2 (0.20.0)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)